runtime/lua/vim/lsp/util.lua

```diff
@@ -661,7 +661,7 @@ function M.focusable_float(unique_name, fn)
   local bufnr = api.nvim_get_current_buf()
   do
     local win = find_window_by_var(unique_name, bufnr)
-    if win and api.nvim_win_is_valid(win) and not vim.fn.pumvisible() then
+    if win and api.nvim_win_is_valid(win) and vim.fn.pumvisible() == 0 then
```

Comment:
Can we do `local ok, vim_visible = pcall(vim.fn.pumvisible)`. There are times where this could be called where `vim.fn` might not be callable.

Reply:
Sure, will do. (Just out of curiosity, in what situations could that happen?)

Reply:
Lemme rephrase this: Can I just use `nvim_call_function` instead and circumvent that?

Reply:
ah, no. You want to read `:help api-fast` for more information.